configs/configgen.py

```diff
@@ -49,11 +49,15 @@
 external_virtual_hosts = [
 {
     'name': 'dynamodb_iad',
-    'address': "tcp://127.0.0.1:9204",
+    'address': "127.0.0.1",
+    'protocol':"TCP",
+    'port_value':"9204",
```

Comment:
nit: inconsistent spacing. let's run `autopep8` over the python files to get a consistent format